Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean

```diff
@@ -460,7 +454,7 @@ theorem SpecΓIdentity_naturality {R S : CommRingCat} (f : R ⟶ S) :
     (Scheme.Spec.map f.op).1.c.app (op ⊤) ≫ SpecΓIdentity.hom.app _ =
       SpecΓIdentity.hom.app _ ≫ f := SpecΓIdentity.hom.naturality f
 
-theorem SpecΓIdentity_hom_app_presheaf_obj {X : Scheme} (U : Opens X) :
+theorem SpecΓIdentity_hom_app_presheaf_obj {X : Scheme.{u}} (U : Opens X) :
```

Comment:
Probably this one is worth stressing. I don't change the proof at all; I just make the universe explicit, and according to the profiler the total time taken on my machine goes from 0.7 secs to 0.23.